app/assets/locales/android_translatable_strings.txt

```diff
@@ -164,7 +164,6 @@ post.io.error=Error reading server response: ${0}
 post.unknown.response=Received unknown resonse code (${0}) from server
 post.client.error=Client-side error (code ${0}) received from network request.
 post.server.error=Server-side error (code ${0}) received from network request.
-post.redirection.error=Network request resulted in redirection (code ${0}). Please use a non-redirecting url.
 post.gone.error=Case is not present on server.

```

Comment:
Shouldn't there be a new string here for the new error case? (Attempt to auth over HTTP)?

Reply:
Currently the error is only getting logged into logs and not shown to user and hence we don't have a corresponding error here. Though just realized there are few other places where PlainTextPasswordException should get caught and logged. Will add it them once we decide on whether to retain the exception or not.

Reply:
I think it's possible that the other requester was just never triggering the error. I think it's important that the user be able to see the specific error if it occurs (since otherwise it will be impossible for an end-user to understand why their network requests are failing), but since the error would actually only ever occur when someone technical is setting up a new environment it actually probably doesn't need to be in the translation set (IE: It's fine if the error isn't localizable and the message is in code, as long as it's clear that the error is "the server you are connecting to is insecure (HTTP), so commcare can't connect with authentication."